src/utils/storage.ts

```diff
@@ -0,0 +1,21 @@
+import Todo from '../ui/components/-utils/todo';
+
+export default class StorageService {
+
+  private storageKey = 'todos-glimmer';
```

Comment:
I realize this is a major nitpick but can you change this to `'glimmer-todomvc:todos'`?